api/models.py

```diff
@@ -145,12 +145,12 @@ class ReportType(models.TextChoices):
 class ContactUs(models.Model):
     email = models.CharField(max_length=50, blank=True)
     name = models.CharField(max_length=50, blank=True)
-    message = models.CharField(max_length=2000, null=True)
+    message = models.TextField()
 
 
 class Feedback(models.Model):
     FEEDBACK_REASON = (("W", "Website"), ("Pp", "Penpal"), ("P", "Performance"))
 
-    reason = models.CharField(max_length=2, choices=FEEDBACK_REASON, null=True)
-    message = models.CharField(max_length=2000, blank=True)
+    reason = models.TextField(choices=FEEDBACK_REASON)
```

Comment:
This was fine as a `CharField` because it is only meant to store the two-character feedback reason. You might run into issues when you generate migrations if you change from `null=True` to `null=False` (defaults to `null=False`).
```suggestion
    reason = models.CharField(max_length=2, choices=FEEDBACK_REASON)
```